pdfminer/converter.py

```diff
@@ -177,8 +181,17 @@ def render(item):
             elif isinstance(item, LTImage):
                 if self.imagewriter is not None:
                     self.imagewriter.export_image(item)
+            elif isinstance(item, LTLine):
+                self.write_text("-----\n")
```

Comment:
The line length is fixed atm, but if an equation is very large, the line will not scale with its size.

Reply:
Yes, this is because it is diffucult to know how long the line should be in the conosle. We do have access to how wide the line was in the pdf but converting that width into characters in the console is realy tricky and tedious. So for now it is like this